src/test/java/net/sf/jabref/logic/util/strings/StringUtilTest.java

```diff
@@ -186,4 +186,41 @@ public void testBooleanToBinaryString() {
         assertEquals("1", StringUtil.booleanToBinaryString(true));
     }
 
+    @Test
+    public void testIntValueOf() {
+        assertEquals(1, StringUtil.intValueOf("1"));
+        assertEquals(1234567890, StringUtil.intValueOf("1234567890"));
+    }
+
+    @Test
+    public void testIntValueOfExceptionLetter() {
+        try {
+            StringUtil.intValueOf("12A2");
+            fail();
+        } catch (NumberFormatException ignored) {
+            // Ignored
+        }
+    }
+
+    @Test
```

Comment:
just add a `@Text(expected = NumberFormatException.class)` to test that a specific exception has to occur. The test itself is only one line long then.